dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/jaxrspropagation/common/PropagationHeaderClient.java

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.microprofile.telemetry.internal_fat.apps.jaxrspropagation.common;
+
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+
+@Path("/propagationHeaderEndpoint")
+public interface PropagationHeaderClient {
+
+    @GET
+    public void get();
```

Comment:
A GET that doesn't get anything??

Reply:
Yep, weird but allowed. I don't need to return anything from this method.

Reply:
I guess also a bit weird since the JAX-RS resource method returns the string "OK", so this could return `String`. They should at least match...

Reply:
Isn't this actually implemented by `PropagationHeaderEndpoint.get(HttpHeaders)` which returns a String?